[Alerting][Docs] Documentation for action variables. (#92417)
elastic/kibana: docs/user/alerting/defining-alerts.asciidoc

When an alert instance matches a condition, the alert is marked as _Active_ and assigned an action group.  The actions in that group are triggered.
When the condition is no longer detected, the alert is assigned to the _Recovered_ action group, which triggers any actions assigned to that group.

> **Run When** allows you to assign an action to an _action group_. This will trigger the action in accordance with your **Notify** setting.

Reviewer: ```suggestion
**Run When** allows you to assign an action to an action group. This will trigger the action in accordance with your **Notify** setting.
```

[role="screenshot"]
image::images/alert-flyout-action-details.png[UI for defining an email action]

[float]
==== Action variables

> Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action. The list of available variables can be accessed using the "add variable" button. While available variables differ by alert type, all alert types pass the following variables:

Reviewer: ```suggestion
Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass alert values at the time a condition is detected to an action. You can access the list of available variables using the "add variable" button. Although available variables differ by alert type, all alert types pass the following variables:
```

Reviewer: I don't see an "add variable" button in the screenshots.  Is it the + icon?

Author: @gchaps Yes, it is the plus icon in the screenshot.
